### PR TITLE
Domain names rework

### PR DESCRIPTION
**Pull request by cnivolle**

### CODE REVIEW

File: commons/admin-console/custom-domain-names.md [hunk 1]
```diff
@@ -11,80 +11,131 @@ keywords:
...
-* Using a ***.cleverapps.io** free domain
+* Using a ***.cleverapps.io** free domain, with built-in SSL
 
-<figure class="cc-content-img"><img  src="/doc/assets/images/domain1.png"></figure>
-<figcaption>Example of domain names customization</figcaption>
+## Linking your Domain Name to your Application
 
-## Personal domain name
 
+You have to declare one or several domain names in the console.
```
Comment (divarvel) [inline commons/admin-console/custom-domain-names.md]: You can declare*

File: commons/admin-console/custom-domain-names.md [hunk 2]
```diff
@@ -11,80 +11,131 @@ keywords:
...
+    <td>``domain.par.clever-cloud.com``</td>
+  </tr>
+  <tr>
+    <td>A<div><small>Only if CNAME is not available</small></div></td>
+    <td>Two records on ``62.210.112.171`` and ``62.210.92.244``</td>
+  </tr>
+</table>
 
-  * Domains like ``example.com``
+<br/>
```
Comment (divarvel) [inline commons/admin-console/custom-domain-names.md]: :-S

File: commons/admin-console/custom-domain-names.md [hunk 3]
```diff
@@ -11,80 +11,131 @@ keywords:
...
+  </tr>
+</table>
 
-### Your application runs in the North-America ('MTL') zone
 
-You have two non exclusive choices to add custom personal domain names:
+<br/>
 
-  * Sub-domains like ``hello.example.com`` and wildcard sub-domains like ``*.example.com``
+Two kinds of records are supported, Type A and CNAME. **The use of a CNAME record is highly recommended.** The CNAME configuration will ensure that your DNS configuration will be up-to-date whereas A records will require an action for you if the IP adresses have to be changed. We monitor all domain names linked to apps, so obsolete or incorrect configurations will be notified by email by our support team to apps owners.
```
Comment (divarvel) [inline commons/admin-console/custom-domain-names.md]: You can point your domain name to clever cloud either with a CNAME record or with A records. **The use of a CNAME record is highly recommended**. With a CNAME record, your DNS configuration is always up-to-date. Using A records will require you to keep the DNS configuration up-to-date manually. Domain names linked to clever cloud applications are monitored, so we will send you an email if your DNS configuration is obsolete or incorrect.

File: commons/admin-console/custom-domain-names.md [hunk 4]
```diff
@@ -11,80 +11,131 @@ keywords:
...
+  <div class="panel-heading">
+     <h4>Warning on CNAME Availability</h4>
+  </div>
+  <div class="panel-body">
+    You cannot use a CNAME  on a top-level domain, or on a subdomain which already has DNS records.
+  </div>
+</div>
 
-  * Domains like ``example.com``
+If you want to redirect your top-level domain name to a subdomain through HTTP (eg ``domain.com`` to ``www.domain.com``), most domain name registrars provide a **web redirection service**.
```
Comment (divarvel) [inline commons/admin-console/custom-domain-names.md]: If you want to make you application available from a domain name which does not support CNAME records (eg ``example.com`` in addition to ``www.example.com``, check if your registrar provides a **web redirection service**. This way, you only have to make ``ẁww.example.com`` point to Clever Cloud. Please note that web redirection provided registrars only work over HTTP.

File: commons/admin-console/custom-domain-names.md [hunk 5]
```diff
@@ -11,80 +11,131 @@ keywords:
...
 
-    On your domain name provider, point two **A records** to ``68.71.34.21`` and ``68.71.34.20``
+Remember that it may take up to a full day for DNS changes to propagate, so be patient. It depend on the TTL of the registrar DNS configuration. To make a faster changes or reverts, most registrars allows you to change the TTL value (around 3 or 4 hours, most of the time).
```
Comment (divarvel) [inline commons/admin-console/custom-domain-names.md]: Remember that DNS changes may take time to propagate (usually a few hours, sometimes up to a day or more). It depends on the TTL setting of your DNS configuration. For faster changes, you can lower the TTL value in advance,  and rise it again afterwards.

File: commons/admin-console/custom-domain-names.md [hunk 6]
```diff
@@ -11,80 +11,131 @@ keywords:
...
 
-  * Domains like ``example.com``
+If you want to redirect your top-level domain name to a subdomain through HTTP (eg ``domain.com`` to ``www.domain.com``), most domain name registrars provide a **web redirection service**.
 
-    On your domain name provider, point two **A records** to ``68.71.34.21`` and ``68.71.34.20``
+Remember that it may take up to a full day for DNS changes to propagate, so be patient. It depend on the TTL of the registrar DNS configuration. To make a faster changes or reverts, most registrars allows you to change the TTL value (around 3 or 4 hours, most of the time). 
 
+Before changing you domain name configuration, you can lower the TTL of your domain name one day before the migration (like 3min). Any changes on your DNS configuration will be applied faster. Then, when everything is configred properly, you can put the default TTL value.
 
 <br/>
```
Comment (divarvel) [inline commons/admin-console/custom-domain-names.md]: see above

File: commons/admin-console/custom-domain-names.md [hunk 7]
```diff
@@ -11,80 +11,131 @@ keywords:
...
+
+<table class="table table-striped">
+  <thead>
+    <tr>
+      <th>Domain Name Use Case</th> <th>CNAME config</th> <th>record A config</th> <th>Web redirection</th>
+    </tr>
+  </thead>
+  <tboby>
+    <tr>
+      <td>``www.myapp.com`` <br> ``myapp.com``</td>
```
Comment (divarvel) [inline commons/admin-console/custom-domain-names.md]: www.example.com / example.com

File: commons/admin-console/custom-domain-names.md [hunk 8]
```diff
@@ -11,80 +11,131 @@ keywords:
...
+      <td>``myapp.com``</td>
+      <td>No CNAME record needed</td>
+      <td>point ``myapp.com`` to the two IP adresses of the selected region</td>
+      <td>No redirect needed</td>
+    </tr>
+  </tboby>
+</table>
+<br>
+
+## Clever Cloud's URLs
```
Comment (divarvel) [inline commons/admin-console/custom-domain-names.md]: you can put it above the DNS config part (and it's not URLs, but domain names)

File: commons/admin-console/custom-domain-names.md [hunk 9]
```diff
@@ -11,80 +11,131 @@ keywords:
...
+
+If you don't have a custom domain name configured, you can still test your app online via the Clever Cloud's sub-domains: ``*.cleverapps.io``. There is a default entry configured by default for every new app: ``yourAppID.cleverapps.io``.
+
+In your application's domain section, just enter ``yourSubdomain.cleverapps.io``. You have to choose a unique one.
+
+Trusted SSL is available on every sub-domains.
```
Comment (divarvel) [inline commons/admin-console/custom-domain-names.md]: Every `cleverapps.io` subdomain is available over HTTPs.

File: commons/admin-console/custom-domain-names.md [hunk 10]
```diff
@@ -11,80 +11,121 @@ keywords:
...
+* Using a ***.cleverapps.io** free domain, with built-in SSL
 
-<figure class="cc-content-img"><img  src="/doc/assets/images/domain1.png"></figure>
-<figcaption>Example of domain names customization</figcaption>
+## Linking your Domain Name to your Application
 
-## Personal domain name
+You have to declare one or several domain names in the console.
 
+Add it in the application configuration: in the console, click on your **application name** in the first panel, then choose **domain names**. You'll have to choice to add a custom domain name or use a sub-domain under ``*.cleverapps.io``
```
Comment (urcadox) [inline commons/admin-console/custom-domain-names.md]: You'll have to choose*

File: commons/admin-console/custom-domain-names.md [hunk 11]
```diff
@@ -11,80 +11,121 @@ keywords:
...
+    You cannot use a CNAME  on a top-level domain, or on a subdomain which already has DNS records.
+  </div>
+</div>
 
-    <a href="http://www.gandi.net" target="_blank">Gandi.net</a> example :
-    <figure class="cc-content-img">
-      <img  src="/doc/assets/images/domain2.png">
-    </figure>
-    <figcaption>CNAME records example</figcaption>
+If you want to make you application available from a domain name which does not support CNAME records (eg example.com in addition to ``www.example.com``, check if your registrar provides a web redirection service. This way, you only have to make ``www.example.com`` point to Clever Cloud. Please note that web redirection provided registrars only work over HTTP.
```
Comment (urcadox) [inline commons/admin-console/custom-domain-names.md]: to make your* application

Comment (urcadox) [inline commons/admin-console/custom-domain-names.md]: There is a missing ')' as well

Comment (urcadox) [inline commons/admin-console/custom-domain-names.md]: web redirections* provided by* registrars

File: commons/admin-console/custom-domain-names.md [hunk 12]
```diff
@@ -11,80 +11,121 @@ keywords:
...
 
-  * Domains like ``example.com``
+Remember that DNS changes may take time to propagate (usually a few hours, sometimes up to a day or more). It depends on the TTL setting of your DNS configuration. For faster changes, you can lower the TTL value in advance, and rise it again afterwards.
 
-    On your domain name provider, point two **A records** to ``68.71.34.21`` and ``68.71.34.20``
+Before changing you domain name configuration, you can lower the TTL of your domain name one day before the migration (like 3min). Any changes on your DNS configuration will be applied faster. Then, when everything is configred properly, you can put the default TTL value.
```
Comment (urcadox) [inline commons/admin-console/custom-domain-names.md]: changing your* domain name

configured*

**Pull request merged**